roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -1218,13 +1567,8 @@ public boolean runOptimize() {
   }
 
   /**
-   * Serialize this bitmap.
-   *
-   * If SERIALIZATION_MODE is set to SERIALIZATION_MODE_PORTABLE, this will rely on the
-   * specification at
-   * https://github.com/RoaringBitmap/RoaringFormatSpec#extention-for-64-bit-implementations.
```

Comment:
```
https://github.com/RoaringBitmap/RoaringFormatSpec#extention-for-64-bit-implementations
As of 0.x, this is **not** the default behavior.
```
seems removed, not just moved. Any good reason?

Reply:
adding a local variable introduce more complex logic here, see {@link #SERIALIZATION_MODE} and {@link #getSerializationMode} for more information about serialization mode.

And the following comment is repeated several times in the comments of the file.
`https://github.com/RoaringBitmap/RoaringFormatSpec#extention-for-64-bit-implementations
As of 0.x, this is **not** the default behavior.`